frontend/src/scenes/session-recordings/player/player-meta/playerMetaLogic.tsx

```diff
@@ -18,18 +26,30 @@ import {
     SessionRecordingPlayerLogicProps,
 } from 'scenes/session-recordings/player/sessionRecordingPlayerLogic'
 
-import { PersonType, PropertyFilterType } from '~/types'
+import { PersonType, PropertyFilterType, SessionRecordingType } from '~/types'
 
 import { SimpleTimeLabel } from '../../components/SimpleTimeLabel'
 import { sessionRecordingsListPropertiesLogic } from '../../playlist/sessionRecordingsListPropertiesLogic'
 import type { playerMetaLogicType } from './playerMetaLogicType'
-
 const recordingPropertyKeys = ['click_count', 'keypress_count', 'console_error_count'] as const
 
 export interface SessionSummaryResponse {
     content: string
 }
 
+function allowListedPersonProperties(sessionPlayerMetaData: SessionRecordingType | null): Record<string, any> {
```

Comment:
we have this fallback to person properties which I'd not used...
ofc the person properties list is mahoosive
which broke the UI
so... let's not send them all